features/0881-vc-sd-jwt-cred-attach/README.md

```diff
@@ -0,0 +1,156 @@
+# Aries RFC 0881: SD-JWT signed W3C Credential (vc+sd-jwt) Attachment format for requesting and issuing credentials
+
+- Authors: George Mulhearn (Anonyome Labs)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2025-12-10
+- Supersedes:
+- Start Date: 2025-12-10
+- Tags: [feature](/tags.md#feature), [protocol](/tags.md#protocol), [credentials](/tags.md#credentials)
+
+## Summary
+
+This RFC registers an attachment format for use in the [issue-credential V2](../0453-issue-credential-v2/README.md) protocol based on W3C credentials with [SD-JWT signatures](https://www.w3.org/TR/vc-jose-cose/#with-sd-jwt) from the [VC Data Model 2.0](https://www.w3.org/TR/vc-data-model-2.0/).
+
+It defines a minimal set of parameters needed to create a common understanding of the verifiable credential to issue. It is based on version [2.0 of the Verifiable Credentials Data Model](https://www.w3.org/TR/vc-data-model-2.0/) which is a W3C recommendation since 15 May 2025.
+
+## Motivation
+
+The Issue Credential protocol needs an attachment format to be able to exchange W3C credentials with SD-JWT signatures.
+
+## Tutorial
+
+Complete examples of messages are provided in the [reference section](#reference).
+
+## Reference
```

Comment:
I'd suggest incorporating some of the request parameters from https://github.com/decentralized-identity/aries-rfcs/tree/main/features/0809-w3c-data-integrity-credential-attachment, to at least allow the wallet to provide the key (with proof) during issuance.

`anoncreds_link_secret` is not needed, but providing it as a `didcomm_signed_attachment` makes sense I think?